dockerfiles/startall.sh

```diff
@@ -51,6 +51,20 @@ dependency_check () {
   fi
 }
 
+dependency_update () {
+
+  if [ -z "$1" ]
+  then
+      CHART_SUBPATH="hpcc"
+  else
+      CHART_SUBPATH=$1
+  fi
+
+  echo -e "\nhelm dependency update ${scriptdir}/../helm/${CHART_SUBPATH}"
+
+  helm dependency update ${scriptdir}/../helm/${CHART_SUBPATH}
```

Comment:
picky: could use eval here, to avoid repeation (and avoiding going out of sync if edited).